cmd/kyma/console/cmd.go

```diff
@@ -27,11 +27,10 @@ func NewCmd(o *Options) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "console",
-		Short: "Opens the Kyma Console in a web browser.",
-		Long: `Use this command to open the Kyma Console in a web browser.
-
-`,
+		Use:        "console",
+		Short:      "Opens the Kyma Console in a web browser.",
+		Long:       `Use this command to open the Kyma Console in a web browser.`,
+		Deprecated: "`console` is deprecated!",
```

Comment:
No "please", please! :D 
I support suggesting what new command to use instead. 

I wouldn't explicitly announce when it is going to be deprecated because if the plans change we'll have to remember to update it here too.

Reply:
Maybe we should mention that it only works on Kyma1?

Reply:
Good idea will do it